lib/inspec/resources/docker_container.rb

```diff
@@ -43,6 +43,16 @@ def running?
       status.downcase.start_with?("up") if object_info.entries.length == 1
     end
 
+    def has_volume?(destination, source)
+      # Todo:
+      # [+] Raise proper exception for containers with no mounted volume instead of saying undefined method `Destination' for nil:NilClass
+      raise Inspec::Exceptions::ResourceFailed, "Could not find any mounted volumes for your container" unless volume_info.Mounts[0]
+
+      is_mounted = false
+      volume_info.Mounts.detect { |mount| is_mounted = mount.Destination == destination && mount.Source == source }
```

Comment:
``` volume_info.Mounts.detect { |mount| mount.Destination == destination && mount.Source == source } ```
The above statement returns either truthy (incase volume is mounted) or nil(incase no such volume is mounted). 

So replacing with the code below to return either true or false:
```
is_mounted = false
volume_info.Mounts.detect { |mount| is_mounted = mount.Destination == destination && mount.Source == source }
is_mounted
```